webapp/channels/src/components/widgets/inputs/input/input.tsx

```diff
@@ -223,7 +230,11 @@ const Input = React.forwardRef((
                 {addon}
             </fieldset>
             {customInputLabel && (
-                <div className={`Input___customMessage Input___${customInputLabel.type}`}>
+                <div
+                    id={errorId}
+                    className={`Input___customMessage Input___${customInputLabel.type}`}
+                    role={customInputLabel.type === ItemStatus.ERROR ? 'alert' : undefined}
```

Comment:
Not sure if we just want to set this to `alert` for all cases as it seems that this is shown when `customInputLabel` is set and the possible values are: `type?: 'info' | 'error' | 'warning' | 'success'` which may all be considered as role`alert`

Reply:
I'd say keep it for error and warning.

Reply:
+1 to Pablo. If we had a separate label and error message, we could leave `role='alert'` on the error message and toggle the visibility of it with `display: none`, but since we use this div for both, we have to change the role